README.md

```diff
@@ -16,8 +16,8 @@ Factory of Redux reducers and their associated actions and selectors.
 `trampss-redux-factory` creates generic reducers, actions and selectors in two lines.
 
 ```es6
-import factory from 'trampss-redux-factory'
-export default factory(/* middlewares */)('id')('api')('todos')
+import { map } from 'trampss-redux-factory'
+export default map({ key: 'id', path: 'api', name: 'todos' })
 ```
 That's it, you exported a reducer function you can register thanks to combinerReducer in Redux.
```

Comment:
What about `That's it, you just exported the reducer function and now you can register it through combinerReducer in Redux.` ?

The reducer function is already exported (this is the code above the sentence)